### ISSUE TITLE
Fiji does not connect

### ISSUE THREAD

**Issue by pwalczysko** [ome/omero-insight#84]
The plugin for Fiji has problems.
The connection fails with differing error messages. Typically, this repeats several times.
The messages can alternate between "wrong username/pwd" and "system failure".

The main problems are on Mac and Linux, although there was a problem also reported when testing on Windows (external user) - but the windows problem has a different message "server not responding"/

Typically, after couple of retries and (maybe) a re-install of the Fiji plugin, the problem goes away.

cc @jburel

**Comment by pwalczysko** [comment]
https://openmicroscopy.slack.com/archives/C1PJHN2D7/p1571653362001400

**Comment by pwalczysko** [comment]
I am now able to repeat every single time on Mac
Cannot really connect using Fiji

**Comment by pwalczysko** [comment]
Sequence (repeatable): (you can ``fix`` the problem using insight or network switch)

CASE 1 - Insight helps:
 - using @manics 's workflow as per https://openmicroscopy.slack.com/archives/C1PJHN2D7/p1571653362001400
 - start Fiji in the Linux docker for first time, connect to outreach, observe that you have a fail

![Screen Shot 2019-10-21 at 14 01 29](https://user-images.githubusercontent.com/2478303/67207592-d6f97100-f40b-11e9-95d4-103c5ed1f3dd.png)


 - close Fiji, start OMERO.insight inside the same docker
 - observe that is succeeds to connect
 - close insight, start Fiji again, try to connect
 - this time, Fiji succeeds to connect

CASE 2-network helps
 - using @manics 's workflow as per https://openmicroscopy.slack.com/archives/C1PJHN2D7/p1571653362001400
 - start Fiji in the Linux docker for first time, connect to outreach, observe that you have a fail
 - close Fiji, switch the network (such as switch the wifi on your laptop off, using now the wire)
 - start Fiji again, try to connect
 - this time, Fiji succeeds to connect

**Comment by pwalczysko** [comment]
Actually, it is enough just to start Fiji the second time in this environment - and it works. But this is not repeating the same situation on Mac... - because there, it is not enough.

**Comment by jburel** [comment]
The error:
```
Ice.ConnectionLost
error = 0
```
when attempting to connect

**Comment by jburel** [comment]
tested with the latest state of master i.e. 3c432d68d57c58b5056001bd634fdd63d1861c00
I can connect without problem

**Comment by pwalczysko** [comment]
the latest state of master (downloaded the merge build from merge-ci) did not work for me.

Note: @manics has on the Linux docker the same behaviour as myself - first time does not work, second time (restart Fiji) it does work. His message is nevertheless "System Failure", mine is "Wrong username/pwd".

It is quite possible that the merge build is fixing something, but not sure whether or not we really have a foolproof Fiji plugin atm.

**Comment by jburel** [comment]
I can see the error
``IceConnectionLost`` the first time but it seems to work fine the second time

**Comment by jburel** [comment]
The problem is coming from the Java version used
* If I use linux bundle with jre from https://imagej.net/Fiji/Downloads. This works fine. The version of Java is 1.8.0_172
* The no-jre version will use the java version installed 1.8.0_222
* @pwalczysko reproduces the problem on mac using the version on mac with jre. This comes with adoptjdk and he can reproduce the issue

It will work the second time on linux

**Comment by pwalczysko** [comment]
I think I have the workflow which repeats the bug on Mac.

  - do not switch off your Mac for the day, but close Fiji, just close the lid, go home, Fiji is not running
  - next morning, start Fiji and try to connect to workshop server
  - bug appears - message is "wrong creds" - but these are always right
  - the bug sometimes goes away after restarting the Fiji and re-trying, but I had also occasions that the problem persisted for several repeats
  - the bug does not seem to appear if the whole Mac is restarted

**Comment by jburel** [comment]
mac: I just installed fiji, installed 5.5.8 plugins. No error

**Comment by ppouchin** [comment]
5.5.8 plugin does not seem to work for Mac users with old MacOS versions (e.g. Yosemite).
Using the no-jre Fiji version with old Java 8 (8.0.77) works, but installing the latest Fiji with adoptjdk doesn't.
Retrying or restarting does not change the outcome. First error messages state "wrong creds", but change later to "system failure". 
Also, nothing is logged to "~/omero/omeroinsight.log" anymore...

**Comment by jburel** [comment]
Thanks @ppouchin 
We have not been able to investigate the source of the problem
It is Fiji related OMERO.insight works

**Comment by ppouchin** [comment]
Yes. I've noticed, although people tend to use the plugin rather than the stand-alone as they can open images directly that way.
Is there a way to see the connection logs for the plugin?

**Comment by jburel** [comment]
logs should go to the fiji log when it is used as a plugin

**Comment by jburel** [comment]
@ppouchin did you notice the issue on newer mac version?

**Comment by ppouchin** [comment]
I only have two Mac users who told me they had this problem: one on Yosemite, the other on El Capitan (10.11.6).

**Comment by NIC-UCSD** [comment]
I'm having this issue running Windows 10 Pro and Java Version 8 Update 201 (build 1.8.0_201-b09). I will check with my user who is having the same problem.

**Comment by evenhuis** [comment]
Hi,

I'm having the same issues

Sometimes I get this error:
![Screen Shot 2020-04-27 at 10 08 28 am](https://user-images.githubusercontent.com/15387514/80325565-0684ac80-8879-11ea-895a-38dc202413cf.png)

sometimes this one
![Screen Shot 2020-04-27 at 10 16 03 am](https://user-images.githubusercontent.com/15387514/80325580-169c8c00-8879-11ea-9b9c-2d1f5f3fcc2e.png)

The Fiji plugins works on windows computers I've tested on. 

Here's my OS/ java version:

### OS 
macOS Mojave 10.14.6 (18G4032)

### Fiji 
2.0.0-rc-71/1.52p
Build ea9a94b5e9

### Java
java version "1.8.0_191"
Java(TM) SE Runtime Environment (build 1.8.0_191-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.191-b12, mixed mode)

Cheers,

Chris
